recipes/p4p/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "p4p" %}
+{% set version = "3.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mdavidsaver/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 2112c7fb851c80cd7b9b8ed065265a510fef1376c0aaf9e1e8fbdeca77b7e2c7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

Reply:
why `--no-deps` is not recommended? I just want to make sure that all my deps are coming from conda-forge and if something changes at the package upstream I can catch it here.

Reply:
It is not that, it is because this option will have no effect on ``conda-build``. Because of that, you can remove it.
I hope I clarified your question! :)